src/Android/Xamarin.Android/Samples/Geometry/FormatCoordinates/FormatCoordinates.cs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Drawing;
+
+namespace ArcGISRuntimeXamarin.Samples.FormatCoordinates
+{
+    [Activity]
+    public class FormatCoordinates : Activity
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+        private EditText _DecimalDegreesEditText;
+        private EditText _DmsEditText;
+        private EditText _UtmEditText;
+        private EditText _UsngEditText;
+
+        // Hold last edited field
+        private EditText _lastEdited;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Format coordinates";
+
+            // Create the UI, setup the control references and execute initialization
+            CreateLayout();
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create the map
+            _myMapView.Map = new Map(Basemap.CreateStreets());
+
+            // Add the graphics overlay to the map
+            _myMapView.GraphicsOverlays.Add(new GraphicsOverlay());
+
+            // Create the starting point
+            MapPoint startingPoint = new MapPoint(0, 0, SpatialReferences.WebMercator);
+
+            // Update the UI with the initial point
+            UpdateUiFromMapPoint(startingPoint);
+
+            // Subscribe to map tap events to enable tapping on map to update coordinates
+            _myMapView.GeoViewTapped += (sender, args) => { UpdateUiFromMapPoint(args.Location); };
+        }
+
+        private void InputTextChanged(object sender, EventArgs e)
+        {
+            // Keep track of the last edited field
+            _lastEdited = (EditText)sender;
+        }
+
+        private void ProcessTextChange(object sender, EventArgs e)
```

Comment:
Seems like the code originally tried to convert the coordinates on text change for the inputs. I think this needs a better name now that it happens as a button click: `UpdateCoordinates`, `UpdateMapPoint`, `RecalculateLocation` e.g.